lib/taskana-core/src/main/java/pro/taskana/task/api/models/Task.java

```diff
@@ -65,100 +67,106 @@
   void setDescription(String description);
 
   /**
-   * Sets the Classification key that - together with the Domain from this Task's work basket -
-   * selects the appropriate {@link Classification} for this Task.
+   * Sets the {@linkplain Classification#getKey() key} of the {@linkplain Classification} that -
+   * together with the {@linkplain WorkbasketSummary#getDomain() domain} from the {@linkplain
+   * pro.taskana.workbasket.api.models.Workbasket Workbasket} of the Task - selects the appropriate
+   * {@linkplain Classification} for this Task.
    *
-   * @param classificationKey the classification key for the Task
+   * @param classificationKey the {@linkplain Classification#getKey() key} of the {@linkplain
+   *     Classification} for the Task
    */
   void setClassificationKey(String classificationKey);
 
   /**
-   * Returns the key of the Workbasket where the Task is stored in.
+   * Returns the {@linkplain WorkbasketSummary#getKey() key} of the {@linkplain
+   * pro.taskana.workbasket.api.models.Workbasket Workbasket} where the Task is stored in.
    *
    * @return workbasketKey
    */
   String getWorkbasketKey();
 
   /**
-   * Returns a Map of custom Attributes.
+   * Returns a Map of customAttributes.
    *
-   * @return customAttributes as {@link Map}
+   * @return customAttributes as Map
    */
   Map<String, String> getCustomAttributeMap();
 
   /**
-   * Sets a Map of custom Attributes.
+   * Sets a Map of custom attributes.
    *
-   * @param customAttributes a {@link Map} that contains the custom attributes
+   * @param customAttributes a Map that contains the custom attributes
    */
   void setCustomAttributeMap(Map<String, String> customAttributes);
 
   /**
-   * Returns a Map of Callback info.
+   * Returns the callbackInfo.
```

Comment:
"callbackInfo" or "callback info"?

Reply:
Similar reasoning to "customAttributes".